supplementary_style_guide/style_guidelines/links.adoc

```diff
@@ -33,6 +33,7 @@ Links to upstream sites, such as GitHub, are considered to be external links.
 Deep links can break more frequently and can inadvertently bypass a site's legal notices.
 * Do not use bare URLs for links.
 Bare URLs are unhelpful because they do not provide adequate context about the link target.
+* Do not use URL shorteners to replace full URLs.
```

Comment:
Text lgtm.

Unrelated to your addition - curious though whether by this "External links" section we mean links that are external to RH, or whether we mean just links that aren't xrefs. 

I assume it's the latter, which makes me wonder whether the title "External links" should be changed to something more general. Maybe we have it this way because just "Links" likely wouldn't work since the top-level category is called that. 

Just thinking out loud... :)